app/Presenters/UserPresenter.php

```diff
@@ -197,6 +197,15 @@ public static function dataTableLayout()
                 'visible' => true,
                 'formatter' => 'departmentsLinkObjFormatter',
             ],
+            [
+                'field' => 'department_manager',
+                'searchable' => true,
+                'sortable' => true,
+                'switchable' => true,
+                'title' => trans('general.department') . ' ' . trans('admin/departments/table.manager'),
```

Comment:
We'd need to create a new translation here - "Department manager" can't always be used in that order (or wirth those words) in other languages.

Reply:
Whoops, I completely missed that on my todo. I searched on the Crowdin project but I think I'm unable to add new strings.

Reply:
The Crowdin translation will be automatically created once you add the string to the file in `resources/lang/en-US`. :)

Reply:
(It's set up to use `en-US` as the "base" language, so anything new in those files get propagated across all of the other languages as well, defaulting to English if the key doesn't exist yet, for example, if I haven't pulled down the latest translations.)

Reply:
Ah that's how it works! I got a bit scared off by the "Please do not submit pull requests with language files directly to the repository" in the documentation :)

Will update the PR shortly.